doc/API.md

```diff
@@ -164,7 +164,8 @@ axe.configure({
 	reporter: 'option',
 	checks: [Object],
 	rules: [Object],
-	locale: Object
+	locale: Object,
+	ver: String
```

Comment:
Why `ver` rather than `version`?

Reply:
Discussed during standup today. Proposal is to deprecate `ver` and to use `axeVersion`. We'll support both for a while, but will eventually drop `ver`.

@downzer0 WDYT?

Reply:
Got feedback, yup, we're going with `axeVersion`, and have `ver` be an alternative, only if `axeVersion` isn't set. Please mark `ver` as deprecated in the docs as well.